functions/lib/integration/parsers/order-to-tiny/index.js

```diff
@@ -38,12 +38,12 @@ module.exports = (order, appData, storeId) => {
   if (buyer) {
     const tinyCustomer = {
       codigo: buyer._id,
-      nome: (buyer.corporate_name || ecomUtils.fullName(buyer)).substring(0, 30) ||
+      nome: (buyer.corporate_name || ecomUtils.fullName(buyer).substring(0, 30))?.replace('&', 'e') ||
```

Comment:
```suggestion
      nome: (buyer.corporate_name || ecomUtils.fullName(buyer))?.substring(0, 30).replace('&', 'e') ||
```

Na mudança você moveu o parênteses errado aí...

Reply:
Sua alteração deixaria passar com mais de 20 chars se `buyer.corporate_name` tiver preenchido com mais de 30 chars, fora isso se for checar a string (seu conditional chain) faz mais sentido antes do substring...